src/components/editor/InputLiteral.jsx

```diff
@@ -223,6 +232,7 @@ InputLiteral.propTypes = {
 
 const mapStateToProps = (state, props) => {
   let result = getProperty(state, props)
+  if (result === undefined) result = []
```

Comment:
Could this logic move into the `getProperty` definition? Something like:

```javascript
export const getProperty = createSelector(
  [ inputPropertySelector ],
  (propertyURI) => {
    return propertyURI.items || []
  }
)
```

:question:

Reply:
Like your suggestion; updated in latest commit.